frontend/components/NftCard.tsx

```diff
@@ -0,0 +1,201 @@
+// http://localhost:3000/Explore
```

Comment:
Is this file renamed? From main its called GalleryCard and Explore/index.tsx also refers to GalleryCard

Reply:
Okay so this is a duplicate file so we can just remove it.